src/app/onion/old-dashboard/old-dashboard.component.html

```diff
@@ -54,7 +54,7 @@
           <div *ngIf="learningObjects?.length; else emptyTemplate">
             <div class="column-headers">
               <div>
-                <clark-checkbox *ngIf="!editingChildren" [setValue]="allSelected && selected.size === learningObjects.length" (action)="selectAll()"></clark-checkbox>
+                <clark-checkbox *ngIf="!editingChildren" [value]="allSelected && selected.size === learningObjects.length" (action)="selectAll()"></clark-checkbox>
```

Comment:
Opinionated nitpick here: The value of `[value]` (..?) might be better as a getter on the class to simplify the template and make it more readable.